workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/GameObjectWorldCommandSystem.cs

```diff
@@ -65,8 +64,9 @@ protected override void OnUpdate()
 
                 foreach (var receivedResponse in reserveEntityIdsResponseData.CommandResponses[i].Responses)
                 {
-                    foreach (var worldCommandHandler in worldCommandResponseHandlers)
+                    foreach (var injectable in worldCommandResponseHandlers)
                     {
+                        var worldCommandHandler = (WorldCommands.Requirables.WorldCommandResponseHandler) injectable;
```

Comment:
Could make it read nicer by casting inside the loop: `for (WorldCommands.Requirables.WorldCommandResponseHandler worldCommandHandler in worldCommandResponseHandlers)`

Reply:
Also twice below